<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sceneClass</class>
 <widget class="QMainWindow" name="sceneClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>scene</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>861</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_rankings">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scene.qrc">
       <normaloff>:/Resources/btn_rankings.png</normaloff>:/Resources/btn_rankings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_help">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scene.qrc">
       <normaloff>:/Resources/btn_help.png</normaloff>:/Resources/btn_help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_about">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>310</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scene.qrc">
       <normaloff>:/Resources/btn_about.png</normaloff>:/Resources/btn_about.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scene.qrc">
       <normaloff>:/Resources/btn_start.tif</normaloff>:/Resources/btn_start.tif</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>64</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e27100;&quot;&gt;Breakout&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scene.qrc"/>
 </resources>
 <connections/>
</ui>
